### PR TITLE
Partially unhardcode mx_roadblock

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
attempt to give some help in #3951
#### Describe the solution
Made two map extras - default roadblock, and roadblock_mil, for two types of roadblocks
Made some changes and tweaking in roadblock map, like removing ASRG from them
Also added zombies on police version
#### Describe alternatives you've considered
pass #62407
#### Testing
Maps generates, and looks pretty nice
#### Additional context
The code part were intact, because i am unable to remove it for myself

mx_roadblock
image
mx_roadblock_mil
image

### CODE REVIEW

File: data/json/regional_map_settings.json [hunk 1]
```diff
@@ -729,6 +729,7 @@
           "mx_science": 400,
           "mx_collegekids": 500,
           "mx_roadblock": 1000,
+          "mx_roadblock_mil": 75,
```
Comment (nornagon) [inline data/json/regional_map_settings.json:732]: To match the 1/6 ratio of the hardcoded mx_roadblock, this should be
```suggestion
          "mx_roadblock": 830,
          "mx_roadblock_mil": 170,
```

File: data/json/overmap/map_extras.json [hunk 2]
```diff
@@ -63,7 +63,19 @@
     "type": "map_extra",
     "name": { "str": "Roadblock (Military)" },
     "description": "This road is blocked by military.",
```
Comment (nornagon) [inline data/json/overmap/map_extras.json:65]: Should this be updated?

Comment (GuardianDll) [reply data/json/overmap/map_extras.json:65]: Yeah, just miss this part

File: data/json/regional_map_settings.json [hunk 3]
```diff
@@ -728,7 +728,8 @@
           "mx_military": 50,
           "mx_science": 400,
           "mx_collegekids": 500,
-          "mx_roadblock": 1000,
+          "mx_roadblock": 830,
+          "mx_roadblock_mil": 170,
```
Comment (nornagon) [inline data/json/regional_map_settings.json:732]: you might want to update these in mods which use mx_roadblock, too.

**Comment by GuardianDll** [comment]
Content vere moved to #62407

**Pull request closed without merging**